javascript/packages/cli/src/cli.ts

```diff
@@ -482,6 +482,17 @@ async function test(
   _opts: any,
 ) {
   const opts = program.opts();
+
+  let extension = testFile.slice(testFile.lastIndexOf(".") + 1);
+
+  if (extension !== "zndsl") {
+    console.log(
+      `\n ${decorators.red(
+        "Error:",
+      )} File extension is not correct. Extension for tests should be '.${extension}'.\n`,
```

Comment:
```suggestion
      )} File extension is not correct. Extension for tests should be '.zndsl'.\n`,
```

Small change, since `${extension}` is set to the one provided.